redis.conf

```diff
@@ -1702,5 +1702,20 @@ rdb-save-incremental-fsync yes
 # active-defrag-max-scan-fields 1000
 
 ############################### PMEM CONFIG ###############################
+
+# Redis supports four different Persistent Memory policies:
+#
+# only-pmem: use only DRAM - do not use Persistent Memory
```

Comment:
should be "only-dram".
The same in the next line.

Reply:
:) done